packages/catalog-server/src/test/lib/catalog_test.ts

```diff
@@ -25,6 +25,13 @@ const testPackage1Path = fileURLToPath(
   new URL('../test-packages/test-1/', import.meta.url)
 );
 
+// A set of import, fetch, search tests that use the same data
+const TEST_SEQUENCE_ONE = 'test-data-1';
```

Comment:
Slightly confused by the term "sequence" here, it's actually a partition name? Or is sequence some technical firestore term?

Reply:
It's my term... I'm meaning a sequence of tests that rely on each other. I really want some kind of test nesting.